src/middlewared/middlewared/plugins/network.py

```diff
@@ -96,14 +96,16 @@ class Config:
             strict=True
         ),
         Str('hostname_local', required=True, validators=[Hostname()]),
+        Str('hostname_b', validators=[Hostname()], null=True),
```

Comment:
Can this be empty string ?

Reply:
This was the weird rebase issue I had, but this change has already been merged into master